api_app/connectors_manager/connectors/abuse_submitter.py

```diff
@@ -0,0 +1,14 @@
+from api_app.connectors_manager.connectors.email_sender import EmailSender
+
+
+class AbuseSubmitter(EmailSender):
+    @property
+    def subject(self) -> str:
+        return f"Take down domain {self._job.parent_job.parent_job.observable_name}"
+
+    @property
+    def body(self) -> str:
+        return (
+            f"Domain {self._job.parent_job.parent_job.observable_name} "
+            "has been reported as malicious. We request you to take it down."
+        )
```

Comment:
```suggestion
        return (
            f"Domain {self._job.parent_job.parent_job.observable_name} "
            "has been detected as malicious by our team. We kindly request you to take it down as soon as possible."
        )
```

Reply:
I would also add a part where we greet them (both at the start and end of the mail) cause it's basic courtesy and a customizable parameter that we can populate with the team name. In this way the email can be customized based on which team is using this playbook. (we can use that parameter to sign the email, to talk about our team in the body, ...)
The idea is to make this request more easily acceptable by the providers.